### ISSUE TITLE
lost custom icons and all text from panel, restart cinnamon no longer fixing issue

### ISSUE THREAD

**Issue by techquila** [linuxmint/Cinnamon#4213]
![screenshot from 2015-05-28 16 21 07](https://cloud.githubusercontent.com/assets/1037593/7852692/ea545d4a-0555-11e5-8158-dcc53b2f151f.png)

Using: Debian Jessie (64) | Cinnamon Desktop with Hactivist 1.4 theme and Malys-uniblue icon set

Steps to reproduce:

Fresh install of Debian Jessie 64bit selecting Cinnamon as Desktop Environment

Install Hactivist theme via Themes > get more themes online

Download and install the malys-uniblue icon set.  set these to be the default icons using 'other settings' tab in 'Themes'.

It used to only occur after full-screen or alt-tab and I'd have to restart cinnamon to fix.

now that fix doesn't work and i'm unsure how to go about troubleshooting this further.

Any help appreciated.

**Comment by dalcde** [comment]
Does this happen with other themes?

**Comment by Neowizard** [comment]
Yes, alcde.
I'm using a slightly modified BlackMate (Also a clean Debian Jessie 64bit), and I'm experiencing this issue.
I just stumbled upon a way to consistently reproduce the issue on my Lenovo X201 laptop (with external monitor connected):
- Install RecordMyDesktop (from Debian repositories) and its gtk GUI. 
- Turn on "Outline Capture Area On Screen" feature in the advance options Misc tab of the GUI
- Use the GUI to draw a large record area and this issue will come up. 
Note: Restarting Cinnamon doesn't help until RecordMyDesktop is shut down.
Note 2: The RecordMyDesktop GUI might not be necessary, but I didn't check.

![screenshot from 2015-09-25 12 55 18](https://cloud.githubusercontent.com/assets/2204128/10097883/40144a1e-6385-11e5-872d-e11531616424.png)
![screenshot from 2015-09-25 12 56 17](https://cloud.githubusercontent.com/assets/2204128/10097882/40125650-6385-11e5-9370-f66ce79ecf06.png)

**Comment by Neowizard** [comment]
Well, for future references (since this issue seems to still exist in 2.6.x), I found a workaround that seems to work wonders.
For some reason, changing the font scaling factor and reverting that change made the issue go away completely. I can't reproduce it anymore, after being able to consistently reproduce it for months now, regardless of any other attempt.
I just ran:
`gsettings set org.cinnamon.desktop.interface text-scaling-factor 1.1`
`gsettings set org.cinnamon.desktop.interface text-scaling-factor 1`
and that was it.
For some reason, after I ran it my background picture reverted to the default Debian Jessie one...no idea why.

Related issue, [Bug #3442](https://github.com/linuxmint/Cinnamon/issues/3442), lead me to this workaround.
